omod/src/main/webapp/pages/conditionlist/addCondition.gsp

```diff
@@ -43,15 +43,16 @@
 <div id="condition" ng-app="conditionApp" ng-controller="ConditionController">
     <h2 class="inline">${ui.message('coreapps.conditionui.addNewCondition')}</h2><br/>
 
-    <div class="horizontal">
-    <label>${ui.message('coreapps.conditionui.condition')} </label>
```

Comment:
Any reason why we moved the label from the above line?

Reply:
In order to get the condition , onset date and end date in same alignment.